### PR TITLE
Fix manifest_version error

### PR DESCRIPTION
**Pull request by ldobreira**

### CODE REVIEW

**Comment by ldobreira** [comment]
@googlebot I signed it!

**Review by dotproto — APPROVED** [review]
Thanks for the fix!

**Comment by dotproto** [comment]
Oops, forgot to merge this after I approved it. Can you update this PR by merging master?

**Comment by dotproto** [comment]
Closing as it's been about a month and the submitter has not updated the PR.

**Pull request closed without merging**
